src/views/Main/Main.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import BlogCard from '../../components/BlogCard/BlogCard';
+import { fetchBlogs } from '../../services/blogs';
+
+export default function Main() {
+  const [blogs, setBlogs] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const resp = await fetchBlogs();
+        setBlogs(resp);
+        setLoading(false);
+      } catch (e) {
+        setErrorMessage('An error ocurred. Please refresh the page');
```

Comment:
Really like the try/catch here. Super important for catching errors when applying 3rd party databases.